docs/migrate-from-cluster.md

```diff
@@ -0,0 +1,226 @@
+# Migration documentation
+
+## Disclaimer
+
+Migrating a cluster from [ck8s-cluster](https://github.com/elastisys/ck8s-cluster) to [compliantkubernetes-kubespray](https://github.com/elastisys/compliantkubernetes-kubespray) is **NOT** recommended, only do this if you have no other alternative.
+If you can, recreating a new cluster is always better.
+
+This has only been tested on CityCloud and might not look the same for other cloud providers
+
+## Steps
+
+1. Set up a kubespray cluster config
+
+    ```ShellSession
+    export CK8S_CONFIG_PATH=/path/to/new/config
+
+    bin/ck8s-kubespray init my-new-cluster openstack MYGPGFINGERPRINT
+    ```
+
+1. Configure terraform to your liking, but it must match old cluster.
+
+    Some variables to look out for
+
+    ```ini
+    cluster_name = "my-cluster" # Same as ${prefix_*c} in tfvars.json
+    public_key_path = "/path/to/ssh/key.pub" # Should be ${CK8S_CONFIG_PATH}/ssh/id_rsa_sc.pub
+    network_name = "my-cluster-network" # Same as ck8s-cluster (${prefix_*c}-network)
+    floatingip_pool = "ext-net" # Same as "external_network_name" in tfvars.json
+    external_net = "fba95253-5543-4078-b793-e2de58c31378" # Same as "external_network_id" in tfvars.json
+
+    dns_nameservers = [ "8.8.8.8", "8.8.4.4" ] # Hardcoded in ck8s-cluster
+    subnet_cidr = "172.16.0.0/24" # Hardcoded in ck8s-cluster
+    ```
+
+1. Move necessary state from ck8s-cluster to kubespray
+
+      Save the IDs of the objects you want to move
+
+    ```ShellSession
+    $ pwd
+    <ck8s-cluster repo root>/terraform/citycloud
+
+    export CK8S_CONFIG_PATH=/path/to/old/config
```

Comment:
Maybe include the prompt here like on the first line?
```suggestion
    $ export CK8S_CONFIG_PATH=/path/to/old/config
```

Reply:
Yes, I'm kinda torn on this. I like the "copy'bility" of a line that's without the prompt, but when you have the prompt it's clearer that the command is run. 
But now that I'm looking at it, it looks a bit wierd.
Do you think splitting this into two blocks would be better? Or add the prompt to all the command lines?

Reply:
Yea I was thinking the same. Maybe it would be better to have one `bash` section with the commands that can be copy/pasted and one `ShellSession` to explain the output? But I'm fine with just adding the prompt here. After all, one should really take the time and check every command when doing something like this, so copy/paste is not that important. :wink:

Reply:
This really fell between the chairs for me. But yes, I think that's a good idea